rust/libnewsboat/src/fmtstrformatter/parser.rs

```diff
@@ -45,6 +46,17 @@ fn spacing(input: &str) -> IResult<&str, Specifier> {
     Ok((input, Specifier::Spacing(chr)))
 }
 
+fn center_format(input: &str) -> IResult<&str, Specifier> {
+    let (input, _) = tag("%=")(input)?;
+    let (input, width) = take_while(|chr: char| chr.is_ascii() && (chr.is_numeric()))(input)?;
+    let (input, format) = take(1usize)(input)?;
+
+    let format = format.chars().next().unwrap();
```

Comment:
Please add a comment explaining why this `unwrap()` would never panic. (Panic crashes the program, so we try to avoid those, but sometimes we are reasonably sure a panic can't happen — we write a comment to explain our reasoning, then)

Reply:
I added the comment and thanks for the detailed explanation why the comment is necessary/convention :)